content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -152,6 +152,20 @@ To delete a Quality Gates rule, click the **Delete** icon next to the **Edit** b
 
 {{< img src="ci/delete_quality_gate_rule.png" alt="Delete a Quality Gates rule" style="width:90%;">}}
 
+## Enable GitHub checks creation
```

Comment:
I had this one as plural as generally customers have more than 1 rule so several checks will be created. Do you think it would still be better to have it singular?

Reply:
Yes, I think the singular form is better here. If you wanted to use a plural, a better construction would be "Enable creation of GitHub checks", but the singular form here is fairly common and I think the user will understand what we mean

Reply:
Perfect, thanks! I've committed the change then